<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HealthCareWorkerLandingPage</class>
 <widget class="QWidget" name="HealthCareWorkerLandingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Electronic Vaccination Data System</string>
  </property>
  <widget class="QPushButton" name="btn_hcwLogin">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_hcwSignup">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Signup</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
